streams/src/main/java/org/apache/kafka/streams/Topology.java

```diff
@@ -617,7 +600,9 @@ public synchronized <K, V> Topology addSink(final String name,
      *
      * @throws TopologyException
      *         if the provided processor name is not unique, or
-     *         if a parent processor/source name is unknown
+     *         if a parent processor/source name is unknown or specifies a sink
+     *
+     * @see org.apache.kafka.streams.processor.api.ContextualProcessor ContextualProcessor
```

Comment:
Think we should point this one out.

Did not link to `FixedKeyProcessor` as it's not really useful for the PAPI -- it's more of a DSL think (even if it's part of `api.*` package.